.github/workflows/conformance-k0s.yaml

```diff
@@ -0,0 +1,174 @@
+name: Conformancek0s1.25
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request:
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+  push:
+    branches:
+      - master
+      - ft/master/**
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
```

Comment:
@cilium/github-sec, @cilium/ci-structure  What do you think about the frequency of running this platform test?

My impression is that there is almost no k0s-specific configuration here, so the existing kind checks are likely to catch most regressions before we release a version. That makes me inclined to think that we shouldn't run this workflow for every PR change or merge, but instead either run it once as part of release processes (maybe after branching), or skip the workflow entirely and rely on the k0s community to perform this testing and report any issues observed.

Thoughts?

Reply:
I would agree, assuming that this workflow will have some flakiness eventually. More tests == more potential flakes, especially in the case where we already have coverage, I don't see what value running this would bring.

Reply:
@christarazi @joestringer should i simply drop the k0s conformance test then ?

Reply:
I posted this on Slack to gather more perspectives but I'm not seeing anyone arguing to keep this workflow. So I think yeah maybe we can just delete the conformance test. It's nice to have a way to confirm it works, but the docs should be sufficient to allow us to validate this. Sorry to go back and forth with you on this, but yes please delete it.

Reply:
I'm late to the party, but my opinion is that CI is OK as long as we have someone in the community maintaining the CI. If we do not, then when CI breaks it'll just get disabled because of lack of cycles to maintain.

FWIW, we already have install guides dedicated to specific platforms / distributions (basically everything from this page: https://docs.cilium.io/en/latest/installation/external-toc/), there is no CI for these and it seems to be mostly OK to me.